### ISSUE TITLE
Adipic aciduria

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#6021]
**Preferred term label:** Adipic aciduria

**Synonyms**

**Definition (free text, please give PubMed ID)** Concentration of  Adipic acid in the urine is above the normal range., PMID: 11182288

**Parent term (use hpo.jax.org/app)** Dicarboxylic aciduria HP:0003215

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
https://hpo.jax.org/app/browse/term/HP:0410309
https://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI:30832

I think they are synonyms, i.e., the alpha does not specify anything./

**Comment by pnrobinson** [comment]
See
https://www.ebi.ac.uk/chebi/searchId.do?chebiId=CHEBI:37024
2-aminoadipic acid is a synonym of alpha-Aminoadipic acid

**Comment by pnrobinson** [comment]
We also need 2-oxoadipic aciduria , see https://pubmed.ncbi.nlm.nih.gov/23141293/

**Comment by pnrobinson** [comment]
Looking at this more closely, it seems that Alpha-aminoadipic aciduria (the existing HPO term) is found in some metabolic diseases. Adipic acid (not alpha) is described as " From an industrial perspective, it is the most important dicarboxylic acid: about 2.5 billion kilograms of this white crystalline powder are produced annually, mainly as a precursor for the production of nylon. Adipic acid otherwise rarely occurs in nature, but it is known as manufactured E number food additive E355." In the PMID, the adipic aciduria was appparently caused by Jello consumption. I do not think that this is a medically relevant term therefore, please reopen if there si evidence in pubmed that adipic aciduria is a sign of a specific disease.
